src/components/styles/Icons/consts.ts

```diff
@@ -0,0 +1,8 @@
+import { IconHello } from 'assets/svg';
+
+export const baseIcons = [
```

Comment:
It would be better to create this array dynamically because right now you have to import every single new icon here. If there will be a lot of icons in the project, then it can be problematic. You can get some inspiration from this project: https://gitlab.com/pagepro/proofed-editor-area/-/tree/master/src/components/styles/Assets